api/workers/cluster.go

```diff
@@ -113,6 +112,23 @@ func (instances Instances) describeASG() error {
 		instances[*i.InstanceId] = instance
 	}
 
+	// describe and log every recent ASG activity
+	dres, err := models.AutoScaling().DescribeScalingActivities(
+		&autoscaling.DescribeScalingActivitiesInput{
+			AutoScalingGroupName: aws.String(resources["Instances"].Id),
+		},
+	)
+	if err != nil {
+		return nil
```

Comment:
I believe the thinking here is fail gracefully. That is, logging/describing the ASG activities isn't as critical the lines above it. It allows code like https://github.com/convox/rack/pull/759/files#diff-b4f593c3fe0f755e9e7dfee53404ceb8R47 to carry on.